tests/LANraragi/Plugin/Metadata/Generic.t

```diff
@@ -0,0 +1,28 @@
+use strict;
+use warnings;
+use utf8;
+use Data::Dumper;
+
+use Module::Pluggable require => 1, search_path => ['LANraragi::Plugin::Metadata'];
+
+use Test::More;
+use Test::Deep;
+
+my @required_keywords = qw( author description icon name namespace parameters type version );
```

Comment:
`icon` and `parameters` could probably be moved to the optional keywords array, they're not necessary as far as the server code goes.

Reply:
I was waiting to ask you this. I just made the difference between the hashes to obtain the two lists as an example.